coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 5.6
+    "covered_percent": 38.69
```

Comment:
But this is what it says after I run rspec: `129 / 155 LOC (83.23%) covered.`??

Reply:
@dfaulken idk why the `covered_percent` in `.last_run.json` is 38.69 but after running rspec, it shows up in the terminal as `83.28`